apps/mark-scan/frontend/src/pages/diagnostics/diagnostics_screen.tsx

```diff
@@ -154,6 +184,27 @@ export function DiagnosticsScreen({
           }}
         />
       </Route>
+      <Route path="/pat-input">
+        <PatDeviceCalibrationPage
+          successScreenButtonLabel={appStrings.buttonDone()}
+          successScreenDescription={appStrings.instructionsBmdPatDiagnosticConfirmExitScreen()}
```

Comment:
Since this flow isn't voter-facing, we can use raw strings here (no translations or audio needed)